index.html

```diff
@@ -50,6 +50,264 @@ <h1>I'm Matunda. Glad to see you!</h1>
+              <div class="role">
+                <p>Back End Dev</p> 
+                
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="year">
+                <p>2015</p>
+              </div>
+            </div>
+
+          
+          <div class="summary">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          
+            <ul class="tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+           
+          <div class="action">
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+
+
+      </div>
+    </section>
+    <!--About Me Section-->
+    <section>
+      <div class="about">
+        <div class="top">
+          <h1>About Me</h1>
+          <div class="message">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          </div>
+          <div class="social-media">
+            <p class="social-media-text">LET'S CONNECT</p>
+            <div class="social-media-icons">
+              <ul>
+              <li><a href="#"><img src="./images/twitter icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Linkedin icon.svg"></a></li>
+              <li><a href="#"><img src="./images/M icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Github icon.svg"></a></li>
+              <li><a href="#"><img src="./images/Chill icon.svg"></a></li>
+            </ul>
+            </div>
+           
+          </div>
+          <div class="resume">
+            <!-- <div class="action"> -->
+              <div class="get-resume-btn">
+                <p>Get my resume</p>
+              </div>
+            
+
+          </div>
+        </div>
+        <ul class="skills-list">
+          <div class="languages">
+            <div class="tab-title">
+              
+              <div class="icon">
+                
+                <img src="./images/Icon - Chevron - Down - Larger.svg" alt="" class="drop-down">
+                
+              </div>
+              <li>Languages</li>
+            </div>
+            <div class="language-list">
+              <div class="js-frame">
+                <p>Javascript</p>
+                <img src="./images/Ellipse 1.svg">
+                
+              </div>
+              <div class="js-frame">
+                <p>HTML</p>
+                <img src="./images/HTML.svg">
+                
+              </div>
+              <div class="js-frame">
+                <p>CSS</p>
+                <img src="./images/CSS.svg">
+                
+              </div>
+            </div>
```

Comment:
- [ ] Please wrap the technology in each category (e.g., "JavasScript", "HTML" and "CSS") within a `ul` tag and each of them within a `li` for correct semantics.

Reply:
done as guided